src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/Report.kt

```diff
@@ -369,11 +400,20 @@ class Report : Extension() {
 		}
 	}
 
+	/**
+	 * A quick function for creating timeout embeds after actions have been performed.
```

Comment:
What's with the quick? We have standard embed functions other places and none of them are called quick.

Reply:
Quick as it has in, it has no input to alter any parts about the embed.